net-irc/atheme-services/files/atheme-services.initd

```diff
@@ -0,0 +1,36 @@
+#!/sbin/openrc-run
+# Copyright 1999-2011 Gentoo Foundation
```

Comment:
This timestamp being wrong also typically means `repoman ci` was not used. Make sure the changes are in fact, repoman clean ;)